templates/base.html

```diff
@@ -33,12 +33,12 @@
 
 % if not website.db
 <div class="notification notification-error">{{ _(
-    "We're currently experiencing technical failures. As a result most things don't work. "
-    "Sorry for the inconvenience, we'll get everything back to normal ASAP."
+    "Liberapay is experiencing technical problems. As a result most things won't work. "
+    "It will be fixed as soon as possible."
 ) }}</div>
 % elif db_is_readonly
 <div class="notification notification-error">{{ _(
-    "Liberapay is currently in read-only mode as we are migrating the database. "
+    "Liberapay is in read-only mode during database migration. "
```

Comment:
Liberapay is already referenced, so it follows that the database migration is of the Liberapay database.